contrib/net/http/roundtripper.go

```diff
@@ -45,6 +46,14 @@ func (rt *roundTripper) RoundTrip(req *http.Request) (res *http.Response, err er
 	if rt.cfg.serviceName != "" {
 		opts = append(opts, tracer.ServiceName(rt.cfg.serviceName))
 	}
+	if host, port, err := net.SplitHostPort(url.Host); err == nil {
+		opts = append(opts, tracer.Tag(ext.NetworkDestinationName, host))
+		if portNum, err := strconv.Atoi(port); err == nil {
+			opts = append(opts, tracer.Tag(ext.NetworkDestinationPort, portNum))
+		}
+	} else {
+		opts = append(opts, tracer.Tag(ext.NetworkDestinationName, url.Host))
+	}
```

Comment:
Let's try to keep the happy path to the left by checking `err != nil` instead. (as documented in the [style guidelines](https://github.com/DataDog/dd-trace-go/wiki/Style-guidelines#general-guidelines))